hbase-client/src/main/java/org/apache/hadoop/hbase/exceptions/ClientExceptionsUtil.java

```diff
@@ -81,13 +77,13 @@ public static boolean isSpecialException(Throwable cur) {
    *            RpcThrottlingException
    * @return null if we didn't find the exception, the exception otherwise.
    */
-  public static Throwable findException(Object exception) {
+  public static Throwable unwrapRemoteException(Object exception) {
```

Comment:
I think the old name is better? As the RemoteException has an unwarpRemoteException method, but here we are doing more.